spacy/lang/fr/lex_attrs.py

```
@@ -4,33 +4,37 @@
 
 
-_num_words = set("""
+_num_words = set(
+    """
 zero un deux trois quatre cinq six sept huit neuf dix
 onze douze treize quatorze quinze seize dix-sept dix-huit dix-neuf
 vingt trente quanrante cinquante soixante septante quatre-vingt huitante nonante
 cent mille mil million milliard billion quadrillion quintillion
 sextillion septillion octillion nonillion decillion
-""".split())
+""".split()
```

Ruff (0.12.2)

Consider using a list literal instead of `str.split`

Replace with list literal

(SIM905)